### PR TITLE
[WiP] Patcher-168 Update docs for v0.3.1

### PR DESCRIPTION
**Pull request by techpink**
Preview: https://deploy-preview-852--pensive-meitner-faaeee.netlify.app/patcher/getting-started/

### CODE REVIEW

**Comment by marinalimeira** [comment]
What would be the entrypoint of this in the Docs initial page?

File: _docs-sources/patcher/running-patcher/update.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Patcher Update
+
+The Patcher update command allows you to update some or all of the module dependencies in the current folder and any child folders.
```
Comment (marinalimeira) [inline _docs-sources/patcher/running-patcher/update.md:3]:
```suggestion
The `patcher update` command allows you to update some or all of the module dependencies in the current folder and any child folders.
```

Comment (techpink) [reply _docs-sources/patcher/running-patcher/update.md:3]: I originally put `patcher update` here but then removed it as there are example usages in the subsections below

File: _docs-sources/patcher/getting-started/update-strategies.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  manual_steps_you_must_follow:
+  - instructions_file_path: ~/-infrastructure-live/dev/_global/website-ci-cd-access/README-TO-COMPLETE-UPDATE.md
+```
+
+This is the `README-TO-COMPLETE-UPDATE.md` written into `dev/_global/website-ci-cd-access`:
+
+```markdown
+# custom-iam-entity v0.61.0 -> v0.62.0 (2023.06.05 12:26:30)
+
+Updated dependency custom-iam-entity in /Users/griff/Gruntwork/dogfood-infrastructure-live/dev/_global/website-ci-cd-access/terragrunt.hcl to version v0.62.0, which contains breaking changes. You MUST follow the instructions in the release notes to complete this update safely: https://github.com/gruntwork-io/terraform-aws-security/releases/tag/v0.62.0
```
[the remark this answers is not in the document]
Comment (techpink) [reply _docs-sources/patcher/getting-started/update-strategies.md]: Fixed. Thanks for catching that

File: _docs-sources/patcher/running-patcher/update.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Example usage:
+```
+patcher update dev
+```
+
+After scanning for dependencies, Patcher will show you the 'Modules View'.
+
+If all the dependencies are fully up to date, then Patcher shows a tick in the "Up to date" column.
+
+![Patcher update screenshot showing dependendency that is fully up to date](/img/guides/stay-up-to-date/patcher/patcher-update-overview-futd.png)
```
[the remark this answers is not in the document]
Comment (marinalimeira) [reply _docs-sources/patcher/running-patcher/update.md:20]: Got it, it's ok to leave as it yes for now!

**Comment by marinalimeira** [comment]
This pr moves the url from `patcher/` to `patcher/getting-started/`. Should it maintain `patcher/` as the entrypoint? In the builded version, `patcher/` returns 404

**Comment by josh-padnick** [comment]
@techpink What's the next step here?

**Pull request closed without merging**